src/components/Trade/hooks/useDonationAmountBelowMinimum.tsx

```diff
@@ -0,0 +1,30 @@
+import { thorchainAssetId } from '@shapeshiftoss/caip'
+import { useMemo } from 'react'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import { SwapperName } from 'lib/swapper/api'
+import { RUNE_OUTBOUND_TRANSACTION_FEE_CRYPTO_HUMAN } from 'lib/swapper/swappers/ThorchainSwapper/constants'
+import { selectMarketDataById } from 'state/slices/marketDataSlice/selectors'
+import { useAppSelector } from 'state/store'
+import { selectDonationAmountFiat } from 'state/zustand/swapperStore/amountSelectors'
+import { useSwapperStore } from 'state/zustand/swapperStore/useSwapperStore'
+
+export const useDonationAmountBelowMinimum = () => {
```

Comment:
Pains me that is a hook, I get it since we need the reactivity of both redux and zustand state and derived selectors, but wondering if we could achieve the same without a hook? i.e is there some trick up our sleeve to combine both stores in the context of one selector?

Reply:
hooks pain me in general too but this is definitely the most reactive way to aggregate from both stores, returns a primitive, doesn't maintain any funky state, and isn't going to be composed up into some god hook like we had before. 

i'm ok with this and would prefer this to in the view layer for sure.